src/main/java/io/github/thebusybiscuit/sensibletoolbox/items/MultiBuilder.java

```diff
@@ -318,15 +319,16 @@ private void doBuild(Player player, Block source, Set<Block> actualBlocks) {
             b.setType(source.getType(), true);
         }
 
-        player.setItemInHand(toItemStack());
+        player.getInventory().setItemInMainHand(toItemStack());
```

Comment:
Sorry, this should really be the last mistake I think, other than that it looks all good.
This method can be invoked with the main and off hand, so you need to account for that, you would need to add a new hand parameter to this method for that, but then you could simply do `updateItemStack(...)` or whatever it is called.

Reply:
will take a look later reverted it so it can be merged